auditlog/conf.py

```diff
@@ -45,3 +45,11 @@
 settings.AUDITLOG_DISABLE_REMOTE_ADDR = getattr(
     settings, "AUDITLOG_DISABLE_REMOTE_ADDR", False
 )
+
+# Enable changes_display_dict truncator
+settings.AUDITLOG_TRUNCATE_CHANGES_DISPLAY = getattr(
+    settings, "AUDITLOG_TRUNCATE_CHANGES_DISPLAY", True
+)
+
+# Number of characters at which changes_display_dict property should be shown
+settings.AUDITLOG_TRUNCATE_LIMIT = getattr(settings, "AUDITLOG_TRUNCATE_LIMIT", 140)
```

Comment:
I think we can handle this with only one setting like `AUDITLOG_CHANGE_DISPLAY_TRUNCATE_LENGTH`.

- If the value is `0` -> display nothing
- if the values is > `0` -> truncate
- if the values < `0` -> display all the string

it has to be documented as well.

What do you think?